R/get_routes.R

```diff
@@ -54,11 +54,12 @@ get_routes = function(od, plans, purpose = "work", folder = ".", batch = TRUE, b
               "-api.cyclestreets.net"
             ),
             pat = Sys.getenv("CYCLESTREETS_BATCH")
-          )  
+          )
+        routes_raw$route_id = routes_raw$route_number
         }
       }
       
-      if(is.character(segments)){
+      if(is.character(segments) && !is(routes_raw, "sf")){
```

Comment:
> Why are you adding this?

Because of changes made by @mem48, the result is not always a list. Adding these lines of code made the build work again. You can test by reverting the change and running the code.